src/Symfony/Bundle/FrameworkBundle/Command/AboutCommand.php

```diff
@@ -26,7 +27,7 @@
  *
  * @final since version 3.4
```

Comment:
letting it `@final` is valid and I think the use case of adding extra info to the `about` output is too